cpmonitor/tests/migrations_test.py

```diff
@@ -0,0 +1,190 @@
+import os
+import json
+from django.apps import apps
+from django.apps.registry import Apps
+from django.db.models import ManyToOneRel
+from django_test_migrations.migrator import Migrator
+
+from cpmonitor.models import Task
+
+app_name = "cpmonitor"
+
+
+def read_fixture(fixture_name: str, app_name: str, migration_state_apps: Apps):
+    """
+    Reads data from a JSON fixture file of the given application into a given applications state.
+
+    This is a drastically simplified version of `django.core.management` "loaddata" command,
+    which cannot be used, since it uses `django.apps.apps` and we need to use `old_state.apps`.
+    """
+    app_path = apps.get_app_config(app_name).path
+    fixture_path = os.path.join(app_path, "fixtures", fixture_name + ".json")
+
+    with open(fixture_path, "rb") as f:
+        json_list = json.load(f)
+        for json_obj in json_list:
+            Model = migration_state_apps.get_model(json_obj["model"])
+            data = {}
+            if "pk" in json_obj:
+                data[Model._meta.pk.attname] = Model._meta.pk.to_python(
+                    json_obj.get("pk")
+                )
+            for field_name, field_value in json_obj["fields"].items():
+                field = Model._meta.get_field(field_name)
+                if field.remote_field:
+                    if not isinstance(field.remote_field, ManyToOneRel):
+                        raise Exception(
+                            "Field type not supported by this migration test."
+                        )
+                    remote_name = field.remote_field.field_name
+                    remote_field = field.remote_field.model._meta.get_field(remote_name)
+                    data[field.attname] = remote_field.to_python(field_value)
+                else:
+                    data[field.name] = field.to_python(field_value)
+            Model.objects.create(**data)
+
+
+def test_complete_migrations_from_0006(migrator: Migrator) -> None:
```

Comment:
It would be nice to test forward and backwards migration in separate tests so that one sees at once which direction is broken

Reply:
Would it be okay, if we put these improvements in a new Issue / PR, since they are code-improving refactorings.

Reply:
Did the name convention thing.

Reply:
> It would be nice to test forward and backwards migration in separate tests so that one sees at once which direction is broken

 Breaking up the test would not work, since the data set is only present in the old version. We would need a second data set which we would have to keep up-to-date... We will need more of these test, anyway, when the data model evolves.